gangplank/cmd/gangplank/pod.go

```diff
@@ -59,13 +67,15 @@ func init() {
 	spec.AddCliFlags(cmdPod.Flags())
 	cmdPod.Flags().BoolVar(&cosaWorkDirContext, "setWorkDirCtx", false, "set workDir's selinux content")
 	cmdPod.Flags().BoolVarP(&cosaViaPodman, "podman", "", false, "use podman to execute task")
+	cmdPod.Flags().StringVar(&cosaViaRemotePodman, "remote-podman", "", "address of the remote podman to execute task")
```

Comment:
I feel like we can get by without a completely new mode here. It seems like we could dynamically detect the `CONTAINER_HOST` env variable OR use a `gangplank pod --podman --remote=ssh://foo@bar/path.sock` and then define `CONTAINER_HOST` from that.

Let's discuss this more high bandwidth and we'll both poke holes in my proposal.

Reply:
Greate idea :)